scripts/qa_crowbarsetup.sh

```diff
@@ -3884,7 +3884,9 @@ function oncontroller_testsetup
         complain 95 could not reach internet
     fi
 
-    if iscloudver 7plus ; then
+    if iscloudver 7plus && crowbar nova proposal show default | rubyjsonparse \
```

Comment:
@dirkmueller I wanted to remember that "this is upgrade" in https://github.com/SUSE-Cloud/automation/pull/1647